pkg/bootstrap/bootstrap_config.go

```diff
@@ -263,7 +263,8 @@ var overrideVar = env.RegisterStringVar("ISTIO_BOOTSTRAP", "", "")
 // WriteBootstrap generates an envoy config based on config and epoch, and returns the filename.
 // TODO: in v2 some of the LDS ports (port, http_port) should be configured in the bootstrap.
 func WriteBootstrap(config *meshconfig.ProxyConfig, node string, epoch int, pilotSAN []string,
-	opts map[string]interface{}, localEnv []string, nodeIPs []string, dnsRefreshRate string) (string, error) {
+	opts map[string]interface{}, localEnv []string, nodeIPs []string, dnsRefreshRate string,
+	sdsEnabled bool, sdsUdsPath, sdsTokenPath string) (string, error) {
```

Comment:
And I now partially agree with @howardjohn  - we can't keep adding parameters to method.

We actually have a structure - ProxyConfig - where we should actually add pretty much everything that is passed here.

But in this particular case - we don't need to, the SDS stuff should be set in the "opts" when this is called.